src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Lottery {
```

Comment:
Extending `Lottery` in `ColorSupplier` seems unnecessary. The `ColorSupplier` class should be independent and focused on providing colors, as per the task requirements.